h2/test_h2_perf.py

```diff
@@ -0,0 +1,98 @@
+"""
+TLS perf tests - load Tempesta FW with multiple TLS handshakes.
+"""
+
+from framework import tester
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2020 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+NGINX_CONFIG = """
+pid ${pid};
+worker_processes  auto;
+
+events {
+    worker_connections   1024;
+    use epoll;
+}
+
+http {
+    keepalive_timeout ${server_keepalive_timeout};
+    keepalive_requests ${server_keepalive_requests};
+    sendfile         on;
+    tcp_nopush       on;
+    tcp_nodelay      on;
+
+    open_file_cache max=1000;
+    open_file_cache_valid 30s;
+    open_file_cache_min_uses 2;
+    open_file_cache_errors off;
+
+    # [ debug | info | notice | warn | error | crit | alert | emerg ]
+    # Fully disable log errors.
+    error_log /dev/null emerg;
+
+    # Disable access log altogether.
+    access_log off;
+
+    server {
+        listen        ${server_ip}:8000;
+
+        location / {
+            add_header Connection keep-alive;
+            root ${server_resources};
+        }
+        location /nginx_status {
+            stub_status on;
+        }
+    }
+}
+"""
+
+TEMPESTA_CONFIG = """
+listen 443 proto=h2;
+srv_group default {
+    server ${server_ip}:8000;
+}
+vhost default {
+    tls_certificate ${tempesta_workdir}/tempesta.crt;
+    tls_certificate_key ${tempesta_workdir}/tempesta.key;
+
+    proxy_pass default;
+}
+"""
+
+class TLSPerf(tester.TempestaTest):
+    clients = [
+        {
+            'id' : 'tls-perf',
+            'type' : 'external',
+            'binary' : 'tls-perf',
+            'cmd_args' : (
+                '-l 1000 -t 2 -T 75 ${server_ip} 443'
```

Comment:
Same as in the previous PR: duration don't need to be hardcoded and can be taken from the configuration time